src/app.ts

```diff
@@ -849,6 +850,13 @@ router.get(
   safeRequestHandler(accountOpeningRequestAPI.retrieveAccountOpeningRequest)
 );
 
+// Business
+
+router.post(
+  "/businesses/:business_id/documents",
```

Comment:
doesn't we need this one for having req.file?
```suggestion
  "/businesses/:business_id/documents",
  fileUpload.single("file"),
```